scripts/install-dependency.sh

```diff
@@ -147,9 +148,43 @@ install_cmake () {
 
 install_version=$2
 install_path=$3
+
+compiler_toolset=$4
+if [[ -z $compiler_toolset ]]; then
+    compiler_toolset=gcc
+fi
+
+if [[ "$CXX_STANDARD" == 17 ]]; then
+    echo "Install dependency with C++17 flag requested"
+    cmake_cxx_standard_option=-DCMAKE_CXX_STANDARD=17
```

Comment:
is this variable used anywhere?  codacy is showing a warning about it

Reply:
It was for any dependencies using cmake as their build system - but that was just zeromq, but it didn’t like getting compiled with c++17, and is being used as a shared library anyway.